src/install.ts

```diff
@@ -21,12 +21,12 @@ export default function install(): void {
 		tooltipContent: t('Add comment to selected text.'),
 		keyBinding: 'ctrl+alt+m',
 		osxKeyBinding: 'cmd+alt+m',
-		enabledSelector: 'not(fonto:remote-document-id(fonto:selection-common-ancestor()) = "clogs/allannotationsdisabled.xml")',
+		enabledSelector: 'fonto:selection-common-ancestor() and not(fonto:remote-document-id(fonto:selection-common-ancestor()) = "clogs/allannotationsdisabled.xml")',
```

Comment:
Why did you add `exists` to all the others except for this one? If not needed, can you remove from the others?